var/spack/repos/builtin/packages/sse2neon/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import glob
+
+
+class Sse2neon(Package):
+    """A C/C++ header file that converts Intel SSE intrinsics to ARN NEON
+    intrinsics."""
+
+    homepage = "https://github.com/jratcliff63367/sse2neon"
+    git      = "https://github.com/jratcliff63367/sse2neon.git"
```

Comment:
Would you consider to take the maintained fork of `SSE2NEON`?  https://github.com/DLTcollab/sse2neon

Reply:
Well, I'm not quite understand, do you have any suggestion or reason for that?

Reply:
The package racon already provides [aarch64.patch](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/racon/aarch64.patch), which was taken from [DLTcollab/sse2neon](https://github.com/DLTcollab/sse2neon). It would be great to unify SSE2NEON with single source since [DLTcollab/sse2neon](https://github.com/DLTcollab/sse2neon) is actively developed so far.

Reply:
Is that https://github.com/DLTcollab/sse2neon can be merged into the upstream?
I think racon can change to `jratcliff63367/sse2neon` too?
I'm afraid that there will be some feature conflict between `jratcliff63367/sse2neon` and `DLTcollab/sse2neon`

Reply:
[DLTcollab/sse2neon](https://github.com/DLTcollab/sse2neon) is arguably the most active fork of existing SSE2NEON derived projects. More intrinsics and bug-fixes were landed. However, `jratcliff63367/sse2neon` no longer updates.

Reply:
@adamjstewart  What's your opinion?

Reply:
@jserv
Can you contact the upstream maintainer? maybe he has some suggestion for that?

Reply:
There is no exact "upstream" since various forks exists as the continuous work for SSE transition. Spack community can benefit from `DLTcollab/sse2neon` because 1) it provides more intrinsics with bug-fixing and Arm/Aarch64 improvements; 2) it has more contributors and recent use cases; 3) its test suite enables more comprehensive coverage;

Reply:
Quote from https://github.com/jratcliff63367/sse2neon
> This project is now deprecated!
> Other software engineers have been making continued improvements on this project in another GitHub repot.
> This is NO LONGER considered the official depot for SSE2NEON!!
> You should please be redirected to:
> https://github.com/DLTcollab/sse2neon

In other words, `DLTcollab/sse2neon` is the official repository now.

Reply:
That's OK, I will modify `sse2neon` and `racon` package if I have enough time.
Or you can make a PR too?

Reply:
@darmac, please check the pull request: https://github.com/spack/spack/pull/20647